python-project/mortar/video.py

```diff
@@ -40,20 +49,31 @@ def extract_frames() -> None:
     """
     For each of the mkv files in the dataset, extract png images from the file
     at a rate of 1 image per second.
+    Reads optional terminal argument `-i/--input INPUT`.
+    Assumes that videos must be in an `INPUT/jp` subfolder.
     """
 
-    files = _files()
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-i', '--input',
+        help='input folder to look for video files in. Videos must be in'
+             '`INPUT/jp` subfolder')
+    args = parser.parse_args()
+    input = args.input
+
+    files = _files(input)
 
     mkv_files = list(filter(lambda x: x.suffix == '.mkv', files))
 
-    for file in mkv_files[0:1]:
+    for file in mkv_files:
         parent = file.parent
         out_dir = Path(parent, 'png')
         out_template = Path(out_dir, f'{file.stem}-%02d.png')
 
         makedirs(out_dir, exist_ok=True)
 
-        command = ['ffmpeg', '-i', file, '-r', '1', out_template]
+        command = ['ffmpeg', '-i', file, '-r', '1',
+                   out_template, '-loglevel', 'error']
```

Comment:
What's the rationale for using the `-loglevel` argument to ffmpeg?

Reply:
error loglevel only outputs actual errors to stdout, whereas a success would be silent